src/endpoints/grpc/grpc.h

```diff
@@ -2,6 +2,7 @@
 // Licensed under the Apache 2.0 License.
 #pragma once
 
+#include "ccf/endpoint_context.h"
```

Comment:
Doesn't look like this include is doing anything? Or is it required from somewhere else?

Reply:
Yes, this is needed otherwise the linker will error out.